Modules/_sqlite/cursor.c

```diff
@@ -470,9 +469,7 @@ _pysqlite_query_execute(pysqlite_Cursor* self, int multiple, PyObject* operation
     if (!func_args) {
         goto error;
     }
-    if (PyTuple_SetItem(func_args, 0, Py_NewRef(operation)) != 0) {
-        goto error;
-    }
+    PyTuple_SET_ITEM(func_args, 0, Py_NewRef(operation));
```

Comment:
I included it because PyTuple_SET_ITEM is faster (and perfectly safe to use here). Can it stay if we call this PR a "micro-optimisation" iso. "hardening"? :)

Reply:
Can it stay under the new title? Let me know if you want it reverted.

Reply:
Fixed in b5e36cd797741bcdfaacb4b88d7627cd4e813835